src/components/appNavigation/stackNavigation.js

```diff
@@ -77,6 +80,11 @@ class AppView extends Component<AppViewProps, AppViewState> {
         if (!userProperties.isMadeBackup && !userProperties.needAddBackupFeed) {
           await userStorage.userProperties.set('needAddBackupFeed', true)
         }
+        if (this.routeIndex !== navigation.state.index) {
```

Comment:
use onNavigationStateChange in createBrowserApp
see here: https://reactnavigation.org/docs/en/screen-tracking.html
and here: https://github.com/react-navigation/web/search?q=onNavigationStateChange&unscoped_q=onNavigationStateChange

Reply:
onNavigationStateChange - Not supported by our version of companion.

Reply:
then update to latest

Reply:
Updated component. method added and it works